doc/how_to_guides/templates.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Templates
+description:
+type: how_to_guide
+created_at: 2024-11-21
+---
+
+# Templates
+
+## Overview
+As you should know from our documentation their [usage](/usage.md),
+a onesie is already a sort of template. This guide will discuss the
+more specifically customized templates that allow us to define and
+easily re-use elements common among tasks of various generic types.
+
+## Why use templates?
+Templates generally consist of taking the same actions as previous
+onesies and performing them on new sets of arguments. They can
+make it easier for others to review your code by separating out what
+we've done before and what is new about the current effort. In a
+broader sense, they help us all avoid the tedium and error risk that
+can occur when digging up, identifying, and ultimately copying similar
+code from the past and adapting it for the needs of today.
+
+## Usage
+
+### Finding templates to use
+If you suspect that the onesie you're about to write is essentially
+something we've done before (possibly several times), it can be worth
+checking if there's a more specific template already available. As of
+this writing, there are templates for establishing new environment
+configuration variables, creating a new feature access, and creating a
+new permission. You can find all the latest templates in the v2 repo,
+at
+[onesie/templates](https://github.com/watermelonexpress/benchprep-v2/tree/release_candidate/onesie/templates).
```

Comment:
I mentioned this below, but it would be cool if there was a command to list existing templates.

Reply:
I agree! If I get around to making one, I'll be sure to tag you in that PR.